src/pages/PlaylistPage.js

```diff
@@ -20,13 +20,17 @@ import LevelNavigation from '../components/PlaylistPage/LevelNavigation';
 import PlaylistNavigation from '../components/PlaylistPage/PlaylistNavigation';
 import CourseInfo from '../components/PlaylistPage/CourseInfo';
 
-const PlaylistPage = ({params, lessonsByLevel, playlists, t}) => {
+const PlaylistPage = ({params, lessonsByLevel, coursePlaylists, t, showPlaylists}) => {
   const levels = Object.keys(lessonsByLevel);
   const lessonLists = levels.map(level => <LessonList key={level} {...{level}} lessons={lessonsByLevel[level]}/>);
   const filter = <LessonFilter courseName={params.course}/>;
   const jumpTo = levels.length > 0 ? <div><LevelNavigation {...{levels}}/></div> : null;
   const courseInfo = <CourseInfo courseName={params.course}/>;
-  const hasPlaylists = !!Object.keys(playlists).length;
+  const hasPlaylists = !showPlaylists ? null :
```

Comment:
Her går `hasPlaylists` fra å være boolean til å være en `<Row>`. Kunne du ikke droppet hele `hasPlaylists`-variabelen, og så lenger nede der du bruker den, heller gjøre `{showPlaylists ? <Row>...coursePlaylists...</Row> : <Row> .... lessons ...</Row>`? Blir mindre rotete også. Om du føler det er bedre kan jo hver av disse to `<Row>`ene defineres på forhånd, men ser ikke at det er viktig/nødvendig her.